Library/Homebrew/cask/lib/hbc/cli/style.rb

```diff
@@ -11,7 +11,7 @@ def self.help
 
       def run
         install_rubocop
-        system "rubocop", *rubocop_args, "--", *cask_paths
+        system({ "XDG_CACHE_HOME" => HOMEBREW_CACHE }, "rubocop", *rubocop_args, "--", *cask_paths)
```

Comment:
What will `HOMEBREW_ENV_FILTERING` do in this case? We will still override it with `HOMEBREW_CACHE` for RuboCop.